x-pack/plugins/canvas/public/components/sidebar_header/custom_element_modal.tsx

```diff
@@ -65,7 +65,22 @@ export interface Props {
   onCancel: () => void;
 }
 
-export class CustomElementModal extends PureComponent<Props> {
+export interface State {
```

Comment:
No need to export if they aren't being used.  Typically we export State and Props if they're being consumed by `redux`, otherwise I'm not sure they're ever necessary beyond the scope of the component.

Reply:
Ah right, that makes sense. I removed the unused exports in 9df6d65.